scripts/set-bucket.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+// script changes the bucket name set in package.json of the test apps.
+
+const fs = require('fs');
+const path = require('path');
+
+// http mock (lib/mock/http.js) sets 'npg-mock-bucket' as default bucket name.
+// when providing no bucket name as argument, script will set
+// all apps back to default mock settings.
+const bucket = process.argv[2] || 'npg-mock-bucket';
+
+const root = '../test';
+const rootPath = path.resolve(__dirname, root);
+const dirs = fs.readdirSync(rootPath).filter((fileorDir) => fs.lstatSync(path.resolve(rootPath, fileorDir)).isDirectory());
+
+dirs.forEach((dir) => {
+  const pkg = require(`${root}/${dir}/package.json`); // relative path
+
+  // bucket specified as part of s3 virtual host format (auto detected by node-pre-gyp)
+  const keys = ['host', 'staging_host', 'production_host'];
+  keys.forEach((item) => {
```

Comment:
Is there some advantage to creating a constant?
```suggestion
  ['host', 'staging_host', 'production_host'].forEach(item => {
```